src/digisos/informasjon/SoknadUnderBehandlingModal.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import NavFrontendModal from "nav-frontend-modal";
+import {Element, Normaltekst, Systemtittel} from "nav-frontend-typografi";
+import Lenke from "nav-frontend-lenker";
+import {Hovedknapp, Knapp} from "nav-frontend-knapper";
+import styled from "styled-components";
+
+const ModalContent = styled.div`
+    padding: 2rem;
+`;
+
+const StyledSystemtittel = styled(Systemtittel)`
+    margin-bottom: 1rem;
+`;
+
+const Paragraph = styled(Normaltekst)`
+    margin-bottom: 1rem;
+`;
+
+const DineSoknaderKnapp = styled(Knapp)`
+    transform: translateY(-2px);
+    @media screen and (min-width: 680px) {
+        margin-left: 2rem;
+    }
+    @media screen and (max-width: 679px) {
+        margin-top: 1rem;
+    }
+`;
+
+const ButtonRow = styled.div`
+    display: flex;
+    @media screen and (min-width: 680px) {
+        flex-direction: row;
+    }
+    @media screen and (max-width: 679px) {
+        flex-direction: column;
+    }
+`;
+
+export const SoknadUnderBehandlingModal = (props: {
+    isOpen: boolean;
+    onRequestClose: (value: boolean) => void;
+    onPrimaryButtonClick: () => void;
+}) => {
+    return (
+        <NavFrontendModal contentLabel="" isOpen={props.isOpen} onRequestClose={() => props.onRequestClose(false)}>
```

Comment:
contentLabel må avklares før denne kan kan gå i prod.

Generelt: Vi har lagt inn egne typedefinisjoner for NavFrontendModal som ikke lengre stemmer med virkeligheten. Det bør ryddes opp i (inkluderer contentLabel for de andre modalene våre)